gneiss/_summary.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, gneiss development team.
+#
+# Distributed under the terms of the GPLv3 License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+import pandas as pd
+
+
+class RegressionResults():
+    """
+    Summary object for storing regression results.
+    """
+    def __init__(self, stat_results,
+                 feature_names=None,
+                 basis=None):
+        """ Reorganizes statsmodels regression modules.
+
+        Accepts a list of statsmodels RegressionResults objects
+        and performs some addition summary statistics.
+
+        Parameters
+        ----------
+        stat_results : list, sm.RegressionResults
+            List of RegressionResults objects.
+        feature_names : array_like, str
+            List of original names for features.
+        basis : np.array, optional
+            Orthonormal basis in the Aitchison simplex.
+            If this is not specified, then `project` cannot
+            be enabled in `coefficients` or `predict`.
+        """
+        self.feature_names = feature_names
+        self.basis = basis
+        self.results = stat_results
+
+        # obtain pvalues
+        pvals = pd.DataFrame()
+        for i in range(len(self.results)):
+            p = self.results[i].pvalues
+            p.name = self.results[i].model.endog_names
+            pvals = pvals.append(p)
+        self.pvalues = pvals
+
+        # calculate the overall R2 value
+        sse = sum([r.ssr for r in self.results])
```

Comment:
Forgot to mention.

The reason why I'm using `ssr` instead of `ess` and `sse` instead of `ssr` is because I find the statsmodels naming convention really confusing.  In the statistical literature, `ssr` typically stands for "sum of squares regression", not "sum of squares residuals".  And `sse` typically stands for "sum of squares error".

I actually opened up an issue [here](https://github.com/statsmodels/statsmodels/issues/3131).  I'd prefer to keep those variables as is.  I'll add more documentation if necessary.